### PR TITLE
Feat/integrate iam

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { ArlasWuiRootComponent } from './components/arlas-wui-root/arlas-wui-root.component';
+
+
+const routes: Routes = [
+  { path: '', component: ArlasWuiRootComponent},
+  { path: 'callback', redirectTo: '' },
+  { path: 'login', component: LoginComponent },
+  { path: 'register', component: RegisterComponent },
+  { path: 'verify/:id/user/:token', component: VerifyComponent },
+  { path: 'password_forgot', component: ForgotComponent },
```
Comment (QuCMGisaia) [inline src/app/app-routing.module.ts:13]: Shouldn't it be forgot_password or password_forgotten ?

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -18,9 +18,14 @@
  */
 
 import { Routes, RouterModule } from '@angular/router';
-import { ArlasWuiComponent } from './app.component';
+import { LoginComponent, RegisterComponent, ToolkitComponent } from 'arlas-wui-toolkit';
+import { ArlasWuiRootComponent } from './components/arlas-wui-root/arlas-wui-root.component';
 
 export const ROUTES: Routes = [
```
Comment (QuCMGisaia) [inline src/app/app.routes.ts:24]: Is it normal that the routes are declared twice, but do not contain the same paths ?

**Review by mbarbet — APPROVED** [review]

File: src/app/components/configs-list/configs-list.component.html [hunk 3]
```diff
@@ -15,5 +20,6 @@ <h4 class="title">{{'Choose a dashboard to load' | translate}}</h4>
       [style.borderLeftColor]="conf.color">
       {{conf.name}}
     </a>
+    <span style="font-size: 14px;text-align: center;padding-top: 10px;" *ngIf="configurations.length === 0 && listResolved">No dashboard in this organisation</span>
```
Comment (QuCMGisaia) [inline src/app/components/configs-list/configs-list.component.html:23]: Should be a class

File: src/app/components/left-menu/left-menu.component.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
   <div class="arlas-analytics">
     <arlas-analytics-menu [showIndicators]="showIndicators"></arlas-analytics-menu>
     <mat-divider></mat-divider>
-    <ng-container *ngIf="isRefreshAnalyticsButton">
+    <ng-container>
```
Comment (QuCMGisaia) [inline src/app/components/left-menu/left-menu.component.html:5]: The ngIf should be kept

File: src/app/components/left-menu/left-menu.component.ts [hunk 5]
```diff
@@ -150,6 +185,37 @@ export class LeftMenuComponent implements OnInit {
...
+
+    const context = canvas.getContext('2d');
+    const centerX = canvas.width / 2;
+    const centerY = canvas.height / 2;
+    const radius = 16;
+    context.beginPath();
+    context.arc(centerX, centerY, radius, 0, 2 * Math.PI, false);
+    context.fillStyle = '#999';
+    context.fill();
+    context.font = '16px Arial';
```
Comment (QuCMGisaia) [inline src/app/components/left-menu/left-menu.component.ts:205]: Shouldn't it be Roboto to have a uniform font over all the application ?

**Review by QuCMGisaia — APPROVED** [review]

File: src/app/components/configs-list/configs-list.component.html (hunk 3 above)
Comment (MohamedHamouGisaia) [reply src/app/components/configs-list/configs-list.component.html:23]: fixed

File: src/app/components/left-menu/left-menu.component.html (hunk 4 above)
Comment (MohamedHamouGisaia) [reply src/app/components/left-menu/left-menu.component.html:5]: fixed

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (MohamedHamouGisaia) [reply src/app/app-routing.module.ts:13]: fixed

File: src/app/app.routes.ts (hunk 2 above)
Comment (MohamedHamouGisaia) [reply src/app/app.routes.ts:24]: fixed

**Pull request merged**